warehouse/query-core/src/test/java/datawave/query/index/lookup/RangeStreamTest.java

```diff
@@ -60,20 +63,49 @@
     @BeforeClass
     public static void setupAccumulo() throws Exception {
         
+        // just like this shard index i am going to need to
+        // create a metadata table
+        
+        final String METADATA_TABLE = "metadataTable";
```

Comment:
I realize this is a WIP, you can use the static names defined in TableName instead of defining these locally.